lib/history.sh

```diff
@@ -3,7 +3,7 @@ shopt -s histappend # append to bash_history if Terminal.app quits
 
 ## Command history configuration
 if [ -z "$HISTFILE" ]; then
-  HISTFILE=$HOME/.bash_history
+  HISTFILE="$HOME/.bash_history"
```

Comment:
Actually, the quote is not needed here although it is harmless. The right-hand side of variable assignments are not subject to word splitting or pathname expansions.